crawler/build.sbt

```diff
@@ -0,0 +1,43 @@
+name := "newspaper-crawler"
+
+version := "1.0"
+
+scalaVersion := "2.12.1"
```

Comment:
Yes, looks like akka-quartz-scheduler for akka 2.5.x is only available for scala 2.12.x.
I'll switch to 2.12.2.